components/automate-cs-oc-erchef/habitat/config/sys.config

```diff
@@ -160,6 +162,11 @@
         {bulk_fetch_batch_size, {{cfg.api.bulk_fetch_batch_size}}}
     ]},
 
+    {i_browse, [
```

Comment:
Should not add these configs. 
Change this:
`    {ibrowse, [
        {default_max_sessions, 256},
        {default_max_pipeline_size, 1}
    ]},`
    to 
   ```    
    {ibrowse, [
        {default_max_sessions, {{cfg.iBrowse.ibrowse_max_pipeline_size}}},
        {default_max_pipeline_size, {{cfg.iBrowse.ibrowse_max_sessions}}}
    ]}, 
```